config/default/kustomization.yaml

```diff
@@ -3,8 +3,8 @@
 # "wordpress" becomes "alices-wordpress".
 # Note that it should also match with the prefix (text before '-') of the namespace
 # field above.
-namePrefix: cluster-api-provider-packet-
-namespace: cluster-api-provider-packet-system
+namePrefix: capp-
+namespace: capp-system
```

Comment:
Where these changes actually necessary? I'd like to avoid changing the namespace or prefix right now since it would be a breaking change for existing users.

Reply:
no not really, just changed to follow other CAPs and because that was too big 😄 

will revert

Reply:
done, thanks for the review

Reply:
Awesome, thanks! I plan on getting things more in-line with the other providers with the full rebranding efforts later this year.